src/model/footer.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from "zod";
+import groq from "groq";
+
+import { zPortableText } from "./portableText";
+import { zImage } from "./image";
+import { gContentGroupProjection, zContentGroup } from "./contentGroup";
+
+export const zFooterData = z.object({
+    logo: zImage,
+    copyrightDate: z.coerce.date(),
+    copyrightNotice: z.string(),
+    organizationName: z.string(),
+    legalText: zPortableText,
+});
+
+export const gFooterDataQuery = groq`
+*[_id == "organization-singleton"][0]{
```

Comment:
This looks like an invalid query - but it looks like you cracked it with the query you wrote inside the `client` method.

I'd recommend replacing this with the groq query you defined in  `getFooterData` (I've found they're easier to debug when you can view them side by side with the model that parses them). Then you can just import that into the client and pass into the fetch method

Reply:
@danlechambre Please review my latest fixes. I think it is the way that you recommended, now.
thanks for the help